docs/effects/sounds.md

```diff
@@ -0,0 +1,115 @@
+
+Goals
+-----
+
+  * Describe a simple way to implement sounds in Forge
+  * There are many, many versions of `playSound` in Minecraft. Describe what they all do
+
+Terminology
+-----------
+
+  * Sound Event - Something that triggers a sound effect. Examples include "minecraft:block.anvil.hit" or "botania:spreaderFire"
+  * Sound Category - The category of the sound, for example "player" or "block" or simply "master". The sliders in the sound settings GUI represent these categories
+  * Sound File - The literal file on disk that is played, usually an .ogg file.
+
+Sounds.json
+-----------
+
+This json should be located in your base asset directory (src/main/resource/assets/MODID/sounds.json) and indicates to the vanilla resource system what Sound Events you declare and what Sound Files those events use.
+
+A full specification is available [on the wiki], but we highlight the important parts here with an example:
+
+```Json
+{
+  "openChest": {
+    "category": "block",
+    "subtitle": "mymod.subtitle.openChest",
+    "sounds": [ "mymod:openChestSoundFile" ]
+  },
+  "epicMusic": {
+    "category": "record",
+    "sounds": [
+      {
+        "name": "mymod:music/epicMusic",
+        "stream": true
+      }
+    ]
+  }
+}
+```
+
+Underneath the top-level object, each key we have defined corresponds to a Sound Event we want to tell the game about: "mymod:openChest" and "mymod:epicMusic". Note that they are written in the sounds.json without the modid.
+
+Under each event we specify the category of the Sound Event, then a subtitle localization key to be shown to hard of hearing users when the Sound Event is played. 
+
+Finally, we specify the actual Sound Files to be played. Note that the value is an array - if we specify multiple Sound Files then the game will randomly choose one to play whenever the Sound Event is triggered.
+
+The two examples represent two different ways to specify a Sound File. The wiki has precise details, but generally, make sure to use the second form for long Sound Files such as BGM or music discs, because the "stream" argument tells Minecraft to not load the entire Sound File into memory but instead stream it from disk. Using the second form also allows you to specify the volume, pitch, and random weight of that Sound File - again, see the vanilla wiki for precise details.
+
+In either case, you specify the path to your Sound File starting from your "sounds" asset directory. Thus, "mymod:openChestSoundFile" corresponds to the path "assets/mymod/sounds/openChestSoundFile.ogg" and "mymod:music/epicMusic" corresponds to the path "assets/mymod/sounds/music/epicMusic.ogg".
+
+Code Registration
+-----------------
+
+Simply specifying the Sound Events in JSON isn't enough, however. Due to changes in the sound system in 1.9, we must also register Sound Events in code. This very simple to do: 
+
+```Java
+ResourceLocation name = new ResourceLocation("mymod", "openChest");
```

Comment:
I'm not convinced that 'name' is a good variable name for this. It's a resource location, let's keep it at that.